<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fan Controller - Manual Interface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="PlotWidget" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Connection</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Port:</string>
             </property>
             <property name="buddy">
              <cstring>portComboBox</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="portComboBox"/>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Baud Rate:</string>
             </property>
             <property name="buddy">
              <cstring>baudComboBox</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="baudComboBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="disconnectButton">
             <property name="text">
              <string>Disconnect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="connectButton">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
        <zorder></zorder>
        <zorder></zorder>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Values</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>188</width>
           <height>80</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Speed (RPM):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="rpmLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Setpoint:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="spLabel">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1118</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Main</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuMain"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
